examples/cloud_context_profile/main.tf

```diff
@@ -13,22 +13,48 @@ provider "aci" {
   insecure = true
 }
 
+# Import Brownfield Virtual Network in Azure cAPIC
 resource "aci_tenant" "terraform_tenant" {
-  name = "cloudTenant"
+  name = "unmanaged-tenant1"
 }
 
 resource "aci_vrf" "vrf" {
   tenant_dn = aci_tenant.terraform_tenant.id
-  name      = "cloudVrf"
+  name      = "unmanaged-VRF2"
 }
 
+# Azure cloud
 resource "aci_cloud_context_profile" "ctx1" {
   name                     = "cloud_context_profile"
-  description              = "cloud_context_profile created while acceptance testing"
+  description              = "update desc"
   tenant_dn                = aci_tenant.terraform_tenant.id
-  primary_cidr             = "10.0.0.0/16"
-  region                   = "us-west-1"
-  cloud_vendor             = "aws"
+  primary_cidr             = "10.1.0.0/16"
+  region                   = "eastus2"
+  cloud_vendor             = "azure"
```

Comment:
Add an example for cloud_vendor = "AWS" as well

Reply:
starting line 49, there is the example for aws